spec/lib/hackney/income/migrate_uh_agreement_spec.rb

```diff
@@ -72,8 +69,8 @@
 
     it 'nothing get\'s migrated' do
       expect(view_court_cases).not_to receive(:execute)
-      expect(create_informal_agreement).not_to receive(:execute)
-      expect(create_formal_agreement).not_to receive(:execute)
+      expect(create_agreement).not_to receive(:execute)
```

Comment:
wait, create agreement use case doesn't have the `execute` method at all right? 🤔 so 
```suggestion
      expect(create_agreement).not_to receive(:create_agreement)
```
here and everywhere else we're testing for `execute` ? 🤔